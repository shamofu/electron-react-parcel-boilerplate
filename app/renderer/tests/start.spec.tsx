/**
 * @jest-environment jsdom
 */

import * as React from 'react';
import * as Renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';

import Start from '../pages/start';

test('page Start', () => {
  const component = Renderer.create(
    <MemoryRouter><Start /></MemoryRouter>,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
