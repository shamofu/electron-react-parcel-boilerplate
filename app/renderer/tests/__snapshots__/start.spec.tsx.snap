// Jest Snapshot v1, https://goo.gl/fbAQLP

exports['page Start 1'] = `
<div
  style={
    Object {
      "marginTop": 100,
      "textAlign": "center",
    }
  }
>
  <h1>
    Electron + React + Parcel
  </h1>
  <h3>
    <a
      href="/"
      onClick={[Function]}
    >
      Back
    </a>
  </h3>
</div>
`;
